web/components/home/Home.js

```diff
@@ -132,6 +132,9 @@ class Home extends React.Component {
   render() {
     return (
       <div className="homepage overflow-hidden flex jc-center ai-center dir-col">
+      <a className="mlh-trust-badge"  href="https://mlh.io/seasons/na-2019/events?utm_source=na-hackathon&utm_medium=TrustBadge&utm_campaign=2019-season&utm_content=black" target="_blank">
+      <img src="https://s3.amazonaws.com/logged-assets/trust-badge/2019/mlh-trust-badge-2019-blue.svg" alt="Major League Hacking 2019 Hackathon Season" style={{width: '100%' }} />
```

Comment:
1. Maybe you can save the badge to `/assets/` and then import and use it (so our webpage doesn't have to make a round trip to the s3 bucket).
2. We have a utility class for `width: 100%` called [`fill-width`](https://github.com/nwplus/nwhacks2019/blob/45867811c1a7e3c91118331723a3305901f07356/web/styles/utils.sass#L6). You can use it like `className="mlh-trust-badge fill-width"`